### ISSUE TITLE
Add Temporary 2016/2019 hybrid detector with SVT from 2016 but 2019 Ecal and Hodoscope.

### ISSUE THREAD

**Issue by mholtrop** [JeffersonLab/hps-java#464]
This is needed to start testing the trigger. It can probably remain on the branch forever :-)

**Comment by mholtrop** [comment]
I take that back. This should just be merged to master, since the correct 2019 detector is not forthcoming.

**Comment by mholtrop** [comment]
The new detectors, v1 and beta1, crash when running "MCFilterBunches":
```java -DdisableSvtAlignmentConstants -XX:+UseSerialGC -Xmx100m -cp ${HPSJAVA} org.hps.util.FilterMCBunches slic_beta1_detector.slcio   spaced_x2.slcio -e250 
Opened input file slic_beta1_detector.slcio
-----------------------------------------------------------
FieldMap3D 
-----------------------------------------------------------
Using field map local file 'fieldmap/418acm2_10kg_corrected_unfolded_scaled_1.0319.dat'
Reading the field grid from 'fieldmap/418acm2_10kg_corrected_unfolded_scaled_1.0319.dat' ... 
 1 X(mm) 
 2 Y(mm) 
 3 Z(mm) 
 4 BX(1000T) 
 5 BY(1000T) 
 6 BZ(1000T) 
 0 End of Header. Data follows in above format 

 ---> ... done reading 
 ---> assumed the order:  x, y, z, Bx, By, Bz 
 ---> Min values x,y,z: -250.0 -70.0 -1500.0
 ---> Max values x,y,z: 250.0 70.0 1500.0
 Maximum Field strength: 1.0510198900241736 
 ---> The field will be offset by 21.17 0.0 457.2

 ---> Range of values x,y,z: 500.0 140.0 3000.0
-----------------------------------------------------------
May 21, 2019 9:41:32 AM org.lcsim.geometry.compact.converter.HPSTracker2014GeometryDefinition build
INFO: HPSTracker2014v1GeometryDefinition: Constructed 14 geometry objects
May 21, 2019 9:41:32 AM org.lcsim.geometry.compact.converter.HPSTracker2014GeometryDefinition build
INFO: HPSTracker2014v1GeometryDefinition: Constructed 12 module bundles
HPSTracker2014v1JavaBuilder: DONE build JAVA modules
May 21, 2019 9:41:32 AM org.lcsim.detector.converter.compact.HPSEcal3Converter makeSubdetectorDetectorElement
INFO: creating detector element for subdetector Ecal
May 21, 2019 9:41:32 AM org.lcsim.detector.converter.compact.HPSEcal3Converter convert
INFO: converting subdetector Ecal
Exception in thread "main" java.lang.NullPointerException
	at org.lcsim.event.base.SamplingFractionManager.getCorrectedEnergy(SamplingFractionManager.java:34)
	at org.lcsim.event.base.BaseCalorimeterHit.calculateCorrectedEnergy(BaseCalorimeterHit.java:120)
	at org.lcsim.lcio.SIOSimCalorimeterHit.<init>(SIOSimCalorimeterHit.java:68)
	at org.lcsim.lcio.SIOSimCalorimeterHitBlockHandler.addCollectionElements(SIOSimCalorimeterHitBlockHandler.java:23)
	at org.lcsim.lcio.AbstractBlockHandler.readCollection(AbstractBlockHandler.java:39)
	at org.lcsim.lcio.AbstractBlockHandler.readBlock(AbstractBlockHandler.java:32)
	at org.lcsim.lcio.LCIOReader.read(LCIOReader.java:77)
	at org.hps.util.FilterMCBunches.main(FilterMCBunches.java:177)
```
This apparently has something to do with how the detector is "constructed" from the lcdd file?

**Comment by mholtrop** [comment]
I don't understand why, but changing the lines in the compact.xml:
```       <processor type="ScoringTrackerHitProcessor" />        
     </readout>
     <readout name="HodoscopeHits">
-      <segmentation type="GridXYZ" gridSizeX="1.0*mm" gridSizeY="0.0" gridSizeZ="0.0" />
-      <id>system:6,layer:2,x:-8,y:-6</id>
+      <id>system:6,barrel:3,layer:4,ix:4,iy:-3</id>
     </readout>
     <readout name="HodoscopeForeHits">
       <id>system:6,barrel:3,layer:4,module:12,sensor:1,side:32:-2,strip:12</id>
```
This avoid this crash.
It seems that the description in the compact.xml and the code reading the detector did not match. 
Git checksum  dc67f67bb..4cb6e5801

**Comment by mholtrop** [comment]
Merged #485
